contrib/runners/orquesta_runner/tests/unit/test_rerun.py

```diff
@@ -216,10 +219,12 @@ def test_rerun_with_missing_workflow_execution_id(self):
         # Delete the workflow execution.
         wf_db_access.WorkflowExecution.delete(wf_ex_db, publish=False)
 
-        # Manually delete the workflow_execution_id from context of the action execution.
+        # Manually delete the workflow_execution_id from context of the liveaction.
         lv_ac_db1.context.pop("workflow_execution")
         lv_ac_db1 = lv_db_access.LiveAction.add_or_update(lv_ac_db1, publish=False)
-        ac_ex_db1 = execution_service.update_execution(lv_ac_db1, publish=False)
+        # Manually delete the workflow_execution_id from context of the action execution.
+        ac_ex_db1.context.pop("workflow_execution")
+        ac_ex_db1 = ex_db_access.ActionExecution.add_or_update(ac_ex_db1, publish=False)
```

Comment:
Can you provide explanation here?  How come we cannot use execution_service.update_execution here?

Reply:
By the time we come to execution_service.update_execution action is already in completed state [(#L212)](https://github.com/StackStorm/st2/pull/5293/files/3eac2c0092e3708bc1398e173f208b1f3e297e60#diff-456027f186e755cf87449fa1fa7373cdbd63e8faa091067b961987a6cdd0e37eL212) which is why popping of workflow id and and updating the execution object will not work with my changes. That's why I've manually updated the object to remove workflow id.

Reply:
Can you put what you describe here in the comment of the code changes?

Reply:
Added a comment